lib/client.js

```diff
@@ -1171,15 +1171,15 @@ MqttClient.prototype._handleConnack = function (packet) {
   if (packet.properties) {
     if (packet.properties.topicAliasMaximum) {
       if (!options.properties) { options.properties = {} }
-      options.properties.topicAliasMaximum = packet.properties.topicAliasMaximum
+      options.properties.serverTopicAliasMaximum = packet.properties.topicAliasMaximum
     }
     if (packet.properties.serverKeepAlive && options.keepalive) {
       options.keepalive = packet.properties.serverKeepAlive
       this._shiftPingInterval()
     }
     if (packet.properties.maximumPacketSize) {
       if (!options.properties) { options.properties = {} }
-      options.properties.maximumPacketSize = packet.properties.maximumPacketSize
+      options.properties.serverMaximumPacketSize = packet.properties.maximumPacketSize
```

Comment:
you've changed this but where is options.properties.serverMaximumPacketSize referenced?

Reply:
It's not use anywhere, It should be use when client sending message, but I don't know the code enough to do it properly, may be this will better translate it meaning?
```suggestion
      options.serverProperties.maximumPacketSize = packet.properties.maximumPacketSize
```